.rubocop.yml

```diff
@@ -4,6 +4,9 @@ require:
 Metrics/MethodLength:
   Max: 20
 
+Metrics/BlockLength:
+  Max: 90
+
```

Comment:
Rather than changing the max for our entire code base, we can use an excluded here for everything in the specs directory. Since rspec works in blocks, it's very very easy to end up with long blocks so we can consider it an exception to the general rule.

https://stackoverflow.com/questions/36896279/how-to-tell-rubocop-to-ignore-a-specific-directory-or-file#36896588

I think it will be 
```yaml
Metrics/BlockLength:
  Exclude:
    - spec/**/*
```